Sources/ActionServiceCallback.swift

```diff
@@ -0,0 +1,34 @@
+import Foundation
+import RxSwift
+
+public final class ActionServiceCallback {
+  let pipeline: SharedPipeline
+  
+  init(pipeline: PublishSubject<Any>) {
```

Comment:
Since we need to push ActionPairs of different types to this pipeline (e.g. ActionPair<String>, ActionPair<Int> etc.) we need to make it type-erased. Unfortunately Swift type system doesn't allow to use protocols in generics, so Any is the only choice here that I can imagine.